racket/src/cs/rumble/vector.ss

```diff
@@ -5,7 +5,7 @@
     (unless (and (fixnum? n)
                  (fx< n 1000))
       (guard-large-allocation who 'vector n (foreign-sizeof 'void*)))
-    (#2%make-vector n v)]))
+    (#3%make-vector n v)]))
```

Comment:
A leftover change that you didn't mean to commit?

Reply:
I think I was unsure if this change was safe to make (because I was unable to determine what the difference between these was).

Reply:
`#3%make-vector` or `make-vector` in this context is an unsafe `make-vector`. That would be bad, because a should-be-safe `(make-vector #f)` at the Racket level (as implemented by the enclosing function) would crash instead of raising an exception.